Makefile

```diff
@@ -264,6 +264,7 @@ test-docs: docs-container
 
 install-cilium-golang-deps:
 	curl -SsL https://github.com/ksonnet/kubecfg/releases/download/v0.8.0/kubecfg-linux-amd64 > ../../../../bin/kubecfg && chmod +x ../../../../bin/kubecfg
+	go build -o ../../../../bin/protoc-gen-go vendor/github.com/golang/protobuf/protoc-gen-go/main.go
```

Comment:
Wouldn't a go install put this in the correct place? or did I miss a cilium specific bin directory that we're now using in PATH?

Reply:
I can't run `go install` perhaps because it's in a vendor directory :thinking:

Reply:
yeah, it's likely that vendor is only used to resolve dependencies during a build... `go run` is probably doing the right thing (Assuming it's running a full build).